dashboard/test/controllers/data_docs_controller_test.rb

```diff
@@ -26,12 +30,12 @@ class DataDocsControllerTest < ActionController::TestCase
   test_user_gets_response_for :create, params: -> {@test_params}, user: nil, response: :redirect
   test_user_gets_response_for :create, params: -> {@test_params}, user: :student, response: :forbidden
   test_user_gets_response_for :create, params: -> {@test_params}, user: :teacher, response: :forbidden
-  test_user_gets_response_for :create, params: -> {@test_params}, user: :levelbuilder, response: :success
+  test_user_gets_response_for :create, params: -> {{key: 'unique_key'}}, user: :levelbuilder, response: :found
 
-  test_user_gets_response_for :show, params: -> {{key: @data_doc_key}}, user: nil, response: :success
-  test_user_gets_response_for :show, params: -> {{key: @data_doc_key}}, user: :student, response: :success
-  test_user_gets_response_for :show, params: -> {{key: @data_doc_key}}, user: :teacher, response: :success
-  test_user_gets_response_for :show, params: -> {{key: @data_doc_key}}, user: :levelbuilder, response: :success
+  test_user_gets_response_for :show, params: -> {{key: @test_params[:key]}}, user: nil, response: :success
+  test_user_gets_response_for :show, params: -> {{key: @test_params[:key]}}, user: :student, response: :success
+  test_user_gets_response_for :show, params: -> {{key: @test_params[:key]}}, user: :teacher, response: :success
+  test_user_gets_response_for :show, params: -> {{key: @test_params[:key]}}, user: :levelbuilder, response: :success
 
   test 'creating a new data doc writes serialization and redirects to show page with key in URL' do
```

Comment:
I tried to do something like

```
  test 'index page returns docs alphabetized by name' do
    first_name = "Aaaa"
    second_name = "Bbbb"
    third_name = "Cccc"

    third_data_doc = create :data_doc, key: third_name
    first_data_doc = create :data_doc, key: first_name
    second_data_doc = create :data_doc, key: second_name
    get :index
```
and then check the `@body.response` field. But the response field was returning HTML rather than a JSON object. I thought it might have something to do with the `authorize_resource` part requiring a login, but (a) no login is needed (verified manually and through the controller tests above) and (b) signing in levelbuilder doesn't change the response.